packages/pf-codemods/README.md

```diff
@@ -639,6 +621,26 @@ Out:
 />
 ```
 
+### pagination-rename-toggleTemplateProps [(#8134)](https://github.com/patternfly/patternfly-react/pull/8134)
+
+We've renamed the `ToggleTemplateProps` prop and to `PaginationToggleTemplateProps`.
```

Comment:
Following @gitdallas 's comment above regarding how this wasn't a prop and is instead used as a type!
```suggestion
We've renamed `ToggleTemplateProps` to `PaginationToggleTemplateProps`.
```